### ISSUE TITLE
Provide Plunkr 

### ISSUE THREAD

**Issue by martin-mueller-solutions** [KeatonTech/Angular-2-Page-Slider#8]
Hi, would you be able to provide a working plunkr (or anything of the sort) with your module running in angular2?

I checked out your demo, but it's kinda hard to figure out whats wrong with my version. I just tried your module in the latest angular2-version but couldn't get it working. 

Thanks in advance

**Comment by danyyacoub** [comment]
Yes, the demo still uses Directive, which was deprecated in RC6.

**Comment by KeatonTech** [comment]
Yeah, sorry, things are a little out of date. Angular code in particular suffers from very rapid code rot. Unfortunately I'm quite busy right now but this is on my list, I'll see if I can get to it in the next few days -- unless somebody else is willing to take it on :)

**Comment by nukec** [comment]
plunkr for angular4 would be cool. npm page tutorial doesn't really work for me
